index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+        integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>My Portfolio</title>
+</head>
+
+<body>
+    <header class="header">
+        <nav class="navbar container">
+            <p>Olawale Olapetan</p>
+            <div class="">
+                <div class="hamburger">
+
+                </div>
+                <div class="hamburger">
+
+                </div>
+                <div class="hamburger">
+
+                </div>
+
+            </div>
+
+        </nav>
+    </header>
+
+    <section class="main container">
+        <div class="text-heading">
+            <h1>
+                Hey There.
+                I’m Wale
+            </h1>
+        </div>
+        <div class="text-subheading">
+            <h3>
+                I am a Software Developer
+            </h3>
+        </div>
+        <div class="text-body">
+            <p>
+                I can help you build a product , feature or website Look through some of my work and experience! If you
+                like
+                what you see and have a project you need coded,
+                don’t hestiate to contact me.
+            </p>
+        </div>
+
+    </section>
+    <div class="socials container">
+        <img src="images/cat.png" alt="">
```

Comment:
- [ ] Please use background image for your decorations, not the image `img`  for your code to be more semantic.

Reply:
I am sorry, I do not understand the comment above. The images are supposed to be the social media icons and if I implement them as a background image, then it will go to the background of the HTML document.